examples/node/express-app/bin/www

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env node
+
+/**
+ * Module dependencies.
+ */
+
+var app = require('../app');
+var debug = require('debug')('express-app:server');
+var http = require('http');
+var amplitude = require('@amplitude/analytics-node');
+
+/**
+ * Get port from environment and store in Express.
+ */
+
+var port = normalizePort(process.env.PORT || '3000');
+app.set('port', port);
+
+/**
+ * Create HTTP server.
+ */
+
+var server = http.createServer(app);
+
+/**
+ * Listen on provided port, on all network interfaces.
+ */
+
+/**
+ * Start by calling amplitude.init(). This must be done before any event tracking
+ * preferrably in the root file of the project, preferrable before starting your server
+ *
+ * Calling init() requires an API key
+ * ```
+ * amplitude.init(API_KEY)
+ * ```
+ *
+ * Optionally, a config object can be provided. Refer to https://amplitude.github.io/Amplitude-TypeScript/interfaces/Types.BrowserConfig.html
+ * for object properties.
+ */
+amplitude.init('API_KEY');
```

Comment:
Just my idea, feel free to ignore or not. Maybe updating this to `amplitude.init(process.env.API_KEY || 'API_KEY');`, so after cloning the repo, no need to update the code, can probably start with `API_KEY=<api-key> npm start`.

Reply:
Interesting point! I like the idea.